Module not found: Error: Can't resolve '../../package.json' in /node_modules/@opencensus/core/build/src/common'

Please answer these questions before submitting a bug report.

### What version of OpenCensus are you using?
```0.0.19```

### What version of Node are you using?
```12.4.0```

### What did you do?
If possible, provide a recipe for reproducing the error.
I am running the angular universal build for production and I got the error below:
```
ERROR in ./node_modules/@opencensus/core/build/src/common/version.js
Module not found: Error: Can't resolve '../../package.json' in '{PATH}/universal/node_modules/@opencensus/core/build/src/common'
 @ ./node_modules/@opencensus/core/build/src/common/version.js 28:12-41
 @ ./node_modules/@opencensus/core/build/src/index.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/zipkin.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/index.js
 @ ./server.ts
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! angular-io-example@1.0.0 webpack:server: `webpack --config webpack.server.config.js --progress --colors`
npm ERR! Exit status 2
```
### What did you expect to see?
```No such error as above```

### What did you see instead?
```
ERROR in ./node_modules/@opencensus/core/build/src/common/version.js
Module not found: Error: Can't resolve '../../package.json' in '{PATH}/universal/node_modules/@opencensus/core/build/src/common'
 @ ./node_modules/@opencensus/core/build/src/common/version.js 28:12-41
 @ ./node_modules/@opencensus/core/build/src/index.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/zipkin.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/index.js
 @ ./server.ts
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! angular-io-example@1.0.0 webpack:server: `webpack --config webpack.server.config.js --progress --colors`
npm ERR! Exit status 2
```
### Additional context
Add any other context about the problem here.

```
I am using **opencensus** for tracing outbound calls in nodejs with **zipkin** dashboard.
Here are packages added in **package.json** under **dependencies**.
```

```
"@opencensus/exporter-zipkin": "0.0.19",
"@opencensus/nodejs": "0.0.19",
"@opencensus/propagation-b3": "0.0.19",
```
Here is the implementation in `server.ts` file:
```
...
// start opencensus
const oTracing = require('@opencensus/nodejs');
const oPropagation = require('@opencensus/propagation-b3');
const proB3 = new oPropagation.B3Format();
const { ZipkinTraceExporter } = require('@opencensus/exporter-zipkin');
// setup zipkin tracing
function generateExpressTracing() {
  try {
    // Add your zipkin url (ex http://localhost:9411/api/v2/spans)
    // and application name to the Zipkin options
    const configZipkin = {
      "zipkinUrl": "http://localhost:9411/api/v2/spans",
      "appName": "digital-p2p",
      "traceOptions": {
        "samplingRate": 1,
        "bufferSize": 1,
        "bufferTimeout": 1,
        "logLevel": 1
      }
    };
    const zipkinOptions = {
      url: configZipkin.zipkinUrl,
      serviceName: configZipkin.appName
    };
    const zExporter = new ZipkinTraceExporter(zipkinOptions);
    // start tracing
    oTracing.start({
      zExporter,
      propagation: proB3,
      samplingRate: configZipkin.traceOptions.samplingRate,
      bufferSize: configZipkin.traceOptions.bufferSize,
      bufferTimeout: configZipkin.traceOptions.bufferTimeout,
      logLevel: configZipkin.traceOptions.logLevel // `1` show errors, if any
    });
  } catch (_err) {
    console.log('Error cause by opencensus tracing express.');
  }
}
generateExpressTracing();
// end setup opencensus exporter report
...
```

I am running webpack analysis "https://webpack.github.io/analyse/" and here are details:
```
Module not found: Error: Can't resolve '../../package.json' in '{PATH}/universal/node_modules/@opencensus/core/build/src/common'
resolve '../../package.json' in '{PATH}/universal/node_modules/@opencensus/core/build/src/common'
  using description file: {PATH}/universal/node_modules/@opencensus/core/package.json (relative path: ./build/src/common)
    using description file: {PATH}/universal/node_modules/@opencensus/core/package.json (relative path: ./build/package.json)
      no extension
        {PATH}/universal/node_modules/@opencensus/core/build/package.json doesn't exist
      .js
        {PATH}/universal/node_modules/@opencensus/core/build/package.json.js doesn't exist
      .ts
        {PATH}/universal/node_modules/@opencensus/core/build/package.json.ts doesn't exist
      as directory
        {PATH}/universal/node_modules/@opencensus/core/build/package.json doesn't exist
[{PATH}/universal/node_modules/@opencensus/core/build/package.json]
[{PATH}/universal/node_modules/@opencensus/core/build/package.json.js]
[{PATH}/universal/node_modules/@opencensus/core/build/package.json.ts]
 @ ./node_modules/@opencensus/core/build/src/common/version.js 28:12-41
 @ ./node_modules/@opencensus/core/build/src/index.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/zipkin.js
 @ ./node_modules/@opencensus/exporter-zipkin/build/src/index.js
 @ ./server.ts```

It looks like it try to load `package.json` in this `/node_modules/@opencensus/core/build/package.json.js` which does not exist. I might suggest to update the line in ```https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/common/version.ts#L30``` to ```pjson = require('./../../../package.json');```.

--- a/packages/opencensus-core/src/common/version.ts
+++ b/packages/opencensus-core/src/common/version.ts
@@ -14,21 +14,5 @@
  * limitations under the License.
  */
 
-interface Package {
-  version: string;
-}
-
-// Load the package details. Note that the `require` is performed at runtime,
-// which means package.json will be relative to the location of this file.
-// If this file has been compiled, it will be in the `/build` directory, so the
-// package path is relative to that location.  Otherwise, it will be relative
-// to the original .ts file.
-let pjson: Package;
-try {
-  pjson = require('../../../package.json');
-} catch {
-  pjson = require('../../package.json');
-}
-
 // Export the core package version
-export const version: string = pjson.version;
+export const version = '0.0.19';

